### DISCUSSION TITLE
Syncplay 1.7.0 RC1

### DISCUSSION THREAD

**Discussion by Et0h** [Syncplay/syncplay#594] [Announcements]
If you find any issues created by this version then please create an issue and note that you are using RC1.

### Known issues:
* On macOS it might give an error about not being able to check for updates. We'll try to get a v2 for the macOS release to fix this issue.

### Headline additions

- Add room permanence option to server (https://github.com/Syncplay/syncplay/pull/434 by @Assistant with additional work in https://github.com/Syncplay/syncplay/pull/487 by @Et0h and https://github.com/Syncplay/syncplay/pull/492 by @odrling)
- Add dark mode support for Windows (https://github.com/Syncplay/syncplay/pull/483 by Et0h)
- Syncplay now supports five more languages:
  * French (https://github.com/Syncplay/syncplay/pull/455 by @sosie-js)
  * Simplified Chinese (https://github.com/Syncplay/syncplay/pull/500 by @lryanner) - Thanks also to @Zhaodaidai for separately working on a Chinese translation
  * Esperanto (by Tirifto)
  * Finnish (#561 by LINUX-SAUNA with additional work in #563 by @Mikaela)
  * Korean (#591 by @MarongHappy)
- Add playlist-related commands (https://github.com/Syncplay/syncplay/issues/472 by Et0h and https://github.com/Syncplay/syncplay/pull/518 by @rtix):
  * /qa [file/url] - add file or url to bottom of playlist
  * /qas [file/url] - add file or url to bottom of playlist and select it
  * /ql - show the current playlist
  * /qs [index] - select given entry in the playlist
  * /qn - select next entry in the playlist
  * /qd [index] - delete the given entry from the playlist

### Tweaks and bug fixes
* If LANG parameter is set in setup, don’t show language dialog (#460 by @atagulalan)
* Trusted Domains: Allow trusting a domain regardless of HTTP basic auth credentials (#437 by @Tremolo4)
* Allow playlists to be played from network (#419 by @zeroeightysix)
* Fix room name case sensitivity UI issue (#403 by @Et0h)
* Remove redundant help button from dialogs (#403 by @Et0h)
* macOS build: upgrade to Python 3.9 and PySide2 5.15.2 (@albertosottile)
* Remove Encoding from .desktop files as it’s depreciated now and add Keywords entry to .desktop files. (@daniel-123)
* Console: Document setting offset in help (#453 by @luke-jr)
* Add deprecation notice for offset help (#435 by @Et0h)
* Add Windows Store mpv.net path support (#494 by @Et0h)
* Update PT-BR translation (#497 by @nipkownix)
* Fixed server argument max-username-length not working (#506 by @powerjungle)
* Fix disconnected monitor bug (#505 by @Et0h fixing #504 raised by @Zefferis)
* Print timestamp when pausing (#521 by @Kansattica)
* Add Windows Store mpv.net path support (issue #494 raised by @soredake and resolved by @Et0h)
* Correct persistence argument type (#496 by @modwizcode)
* Remove mpv fallback warning spam (@Et0h)
* Fixed permanent rooms on Windows servers (#526 raised by @haz429 and resolved by @Et0h)
* Fixes converting lines to array (#515 by @danfai)
* Made Hungarian chars usable from MPV: ő and ű (#515 by @hunbernd)
* Typo fixes (#540 and #541 by @ViktorOn)
* Linux: Fix dark mode detection by moving to darkdetect v0.7.1 (#557 raised by @ExTBH and resolved by @albertosottile)
* macOS: bundle PySide6, create a universal2 app; and explicitly exclude tkinter (@albertosottile)
* Explicitly pass certifi as trustRoot for Twisted (#554 raised by @BODZy resolved by @albertosottile)
* Don’t detect SMPlayer as mplayer2 (#584 raise by @nikgtasa and resolved by @Et0h)
* Fix localhost connection breaking mpv chat (#545 raised by @Abu-AM and resolved by @Et0h)
* More verbose connection error messages (#565 / #553 raised by @Someone7un7 and resolved by @Et0h)
* Fix mpv quit message (#571 / #578 raised by @jiang-yuan and resolved by @Et0h)
* Log –debug as UTF-8 to fix charmap mapping (#586 by @Et0h)
* Add/fix wildcard matching of trusted subdomains (#587 / #588 raised by @name-snrl and resolved by @Et0h)
* Restore server-side playlist on reconnect (#567 / #590 raised by @soredake and resolved by @Et0h)

<hr /><em>This discussion was created from the release <a href='https://github.com/Syncplay/syncplay/releases/tag/1.7.0-RC1'>Syncplay 1.7.0 RC1</a>.</em>

**Comment by powerjungle** [comment]
Btw just noticed, @luke-jr contribution is the wrong number, this is the correct one: https://github.com/Syncplay/syncplay/pull/435

Actually now that I look at all of them closer, there's many mistakes.

For example "Fix room name case sensitivity UI issue" and "Remove redundant help button from dialogs", both point to https://github.com/Syncplay/syncplay/issues/403, wouldn't it make more sense to point to the commits https://github.com/Syncplay/syncplay/commit/69621ee71042de0ee226a930bfed038ce139b45b and https://github.com/Syncplay/syncplay/commit/0b8c2ddc4b82833c2ec3bcc26673f2b8d5a13e79?

**Comment by Et0h** [comment]
@powerjungle Thanks for reporting these issues - they've been corrected in the [final changelog](https://github.com/Syncplay/syncplay/releases/tag/v1.7.0). If you know of any outstanding mistakes then feel free to report them here.

**Reply by powerjungle** [reply]
Thank you for working it out! The final release looks good so far.

**Comment by luke-jr** [comment]
Deprecating offset is a mistake ;)

I probably won't be upgrading if it gets removed.

**Comment by Et0h** [comment]
Syncplay 1.7.0 has now been released at https://github.com/Syncplay/syncplay/releases/tag/v1.7.0 so this discussion is now closed. If people have further issues please use an existing issue or discussion topic or if none exists create a new one.
